google-api/google_search.py

```diff
@@ -267,14 +267,21 @@ def report(data):
         return
     print(f'Report: {__file__}\n')
     print(f'Config: {CONFIG}\n')
-    if report_stats['dt_build_success']:
+    
+    if (report_stats['dt_build_success'] == None):
+        #variable not changed build not attempted
+        print('Google Search DT not attempted')
+    elif (report_stats['dt_build_success'] == True):
+        #variable changed to True build successful
         print(
             'DT build started at: '
             f'{yvr_dt_start.strftime("%Y-%m-%d %H:%M:%S%z (%Z)")}, '
             f'ended at: {yvr_dt_end.strftime("%Y-%m-%d %H:%M:%S%z (%Z)")}, '
             f'elapsing: {yvr_dt_end - yvr_dt_start}.')
     else:
+        #variable changed to False build failed
```

Comment:
I think we should either say "DT means Derived Table" in comments or replace DT with "Derived Table" in emails. Any new developer will get confused with the abbreviation.